### PR TITLE
refactoring Istio dashboard and add control plane dashboard

### PR DESCRIPTION
**Pull request by piotrkpc**
**Description**
- Refactoring: Grafana dashboards uses `{{` and `}}` within dashboards JSON files which clashes with Helm templates. Escaping them in templates is error prone and time consuming. Keeping dashboards definitions in `files/dashboards` makes it easier to use in templates with `toJson` function.
- Adding control plane dashboards

**Related issue(s)**

https://github.com/kyma-project/kyma/issues/13581

### CODE REVIEW

**Comment by cnvergence** [comment]
Looks proper :) LGTM

File: resources/istio-resources/files/dashboards/control-plane.json [hunk 1]
```diff
@@ -0,0 +1,1751 @@
...
+      "12h",
+      "24h",
+      "2d",
+      "7d",
+      "30d"
+    ]
+  },
+  "timezone": "browser",
+  "title": "Istio Control Plane Dashboard",
+  "tags": ["service-mesh", "kyma"],
```
Comment (cnvergence) [inline resources/istio-resources/files/dashboards/control-plane.json]: Probably this needs to be formatted like in the other dashboards.

**Review by cnvergence — APPROVED** [review]

**Review by cnvergence — APPROVED** [review]

File: resources/istio-resources/files/dashboards/README.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# Istio Dashboards
+Dashboards are form official Istio dashboards with minor changes: https://istio.io/latest/docs/ops/integrations/grafana/#configuration
```
Comment (klaudiagrz) [inline resources/istio-resources/files/dashboards/README.md:2]:
```suggestion
Istio Dashboards in Kyma are based on the official [Istio dashboards](https://istio.io/latest/docs/ops/integrations/grafana/#configuration) with minor modifications:
```

File: resources/istio-resources/files/dashboards/README.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
+ - removing DS_PROMETHEUS Grafana variable
```
Comment (klaudiagrz) [inline resources/istio-resources/files/dashboards/README.md:3]:
```suggestion
 - Removed `DS_PROMETHEUS` Grafana variable
```

File: resources/istio-resources/files/dashboards/README.md [hunk 4]
```diff
@@ -0,0 +1,5 @@
...
+ - removing `__inputs` and `__requires` fields
```
Comment (klaudiagrz) [inline resources/istio-resources/files/dashboards/README.md:4]:
```suggestion
 - Removed **__inputs** and **__requires** fields
```

File: resources/istio-resources/files/dashboards/README.md [hunk 5]
```diff
@@ -0,0 +1,5 @@
...
+ - adding tags: `"tags": ["service-mesh", "kyma"],`
```
Comment (klaudiagrz) [inline resources/istio-resources/files/dashboards/README.md:5]:
```suggestion
 - Added tags: **"tags": ["service-mesh", "kyma"],**
```

**Review by klaudiagrz — APPROVED** [review]

**Review by cnvergence — APPROVED** [review]

**Pull request merged**